src/Elmish.WPF/ViewModel.fs

```diff
@@ -107,37 +107,38 @@ type internal TwoWayBinding<'model> = {
   Set: obj -> 'model -> unit
 }
 
-type internal SubModelBinding<'model, 'msg, 'bindingModel, 'bindingMsg> = {
-  SubModelData: SubModelData<'model, 'msg, 'bindingModel, 'bindingMsg>
-  Vm: ViewModel<'bindingModel, 'bindingMsg> voption ref
+type internal SubModelBinding<'model, 'msg, 'bindingModel, 'bindingMsg, 'bindingViewModel> = {
```

Comment:
How about just `'viewModel`?

Reply:
Because it's specifically the view model that is associated with the `'bindingModel` and `'bindingMsg`. Making it `'viewModel` would seem to associate it with `'model` and `'msg`

Reply:
Fine